assets/javascripts/discourse/components/modal/post-event-invite-user-or-group.js

```diff
@@ -0,0 +1,26 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { extractError } from "discourse/lib/ajax-error";
+
+export default class PostEventInviteUserOrGroup extends Component {
+  @tracked invitedNames = null;
+  @tracked flash = null;
+
+  @action
+  async invite() {
+    try {
+      await ajax(
+        `/discourse-post-event/events/${this.args.model.event.id}/invite.json`,
+        {
+          data: { invites: this.invitedNames || [] },
```

Comment:
```suggestion
          data: { invites: this.invitedNames },
```

that + the previous suggestion – would work, right? 😃